index.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+    integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+  <header id="header">
+    <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" id="header-img"
+      alt="">
+    <nav id="nav-bar">
+      <ul>
+        <li><a class="nav-link" href="#features">Features</a></li>
+        <li><a class="nav-link" href="#how-it-works">How it works</a></li>
+        <li><a class="nav-link" href="#pricing">Pricing</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <form id="form" action="https://www.freecodecamp.com/email-submit">
+      <h1>Handcrafted, home-made masterpieces</h1>
+      <input id="email" type="email" placeholder="Enter you email address"> <br>
+      <input id="submit" type="submit" value="GET STARTED">
+    </form>
+    <section id="features">
+      <div>
+        <i class="fa fa-3x fa-fire"></i>
+        <h1>Premium Materials</h1>
+        <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your
+          purchase.</p>
+      </div>
+      <div>
+        <i class="fa fa-3x fa-truck"></i>
+        <h1>Fast Shipping</h1>
+        <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if
+          you are
+          not satisfied.</p>
+      </div>
+      <div>
+        <i class="fa fa-3x fa-battery-full"></i>
+        <h1>Quality Assurance</h1>
```

Comment:
Please note that it is best practice to have only one `h1` tag on an HTML page. I think you should use other `hx` tags for the different headings in your file.

Reply:
thanks @ademibowale. Is it ok if I use the exact same words you suggested or you would want me to use my own words instead?

Reply:
@Wineshuga its okay you can use your words but make sure you phrase your comment politely and give reason why you make request the changes .